src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -1193,19 +1193,24 @@ def create_vm(cmd, vm_name, resource_group_name, image=None, size='Standard_DS1_
 
 def auto_shutdown_vm(cmd, resource_group_name, vm_name, off=None, email=None, webhook=None, time=None,
                      location=None):
+    from ..lab.aaz.latest.lab.global_schedule import Delete as DeleteSchedule, Create as CreateSchedule
     from azure.mgmt.core.tools import resource_id
-    from azure.mgmt.devtestlabs.models import Schedule
     from azure.cli.core.commands.client_factory import get_subscription_id
     subscription_id = get_subscription_id(cmd.cli_ctx)
     client = _dev_test_labs_client_factory(cmd.cli_ctx, subscription_id)
```

Comment:
Can you pls also remove the dev test labs client definition in `client_factory.py`